sdk/finbourne_identity/utilities/api_client_builder.py

```diff
@@ -115,30 +58,30 @@ def build(cls, api_secrets_filename=None, okta_response_handler=None, api_config
 
         # Use the access token provided if it exists
         if token is not None:
-            # Check that there is an identity_url available
-            cls.__check_required_fields(configuration, ["identity_url"])
+            # Check that there is an api_url available
+            cls.__check_required_fields(configuration, ["api_url"])
             api_token = token
         # Otherwise generate an access token from Okta and use a RefreshingToken going forward
         else:
             # Check that all the required fields for generating a token exist
             cls.__check_required_fields(configuration, [
-                "identity_url",
+                "api_url",
```

Comment:
The environment variable and secrets.json key name are now stored in configkeys.json and this is just referenced as apiUrl in the code.